client-admin/src/components/conversation-admin/seed-comment.js

```diff
@@ -1,55 +1,33 @@
 // Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+/** @jsx jsx */
+
 import React from "react";
 import { connect } from "react-redux";
 import { handleSeedCommentSubmit, seedCommentChanged } from "../../actions";
-import Radium from "radium";
 import _ from "lodash";
-import Button from "../framework/generic-button";
 import strings from "../../strings/strings";
+import { Flex, Box, Text, Button, jsx, Link } from "theme-ui";
 
-const styles = {
-  card: {
-    margin: "10px 20px 10px 20px",
-    backgroundColor: "rgb(253,253,253)",
-    borderRadius: 3,
-    padding: 10,
-    WebkitBoxShadow: "3px 3px 6px -1px rgba(220,220,220,1)",
-    BoxShadow: "3px 3px 6px -1px rgba(220,220,220,1)"
-  },
-  seedForm: {
-    width: "100%",
-    maxWidth: 400,
-    resize: "none",
-    border: "1px solid rgb(200,200,200)",
-    marginBottom: 15,
-    borderRadius: 3,
-    minHeight: 100,
-    fontSize: 16,
-    fontWeight: 300,
-    padding: 10
-  },
-};
-
-@connect(state => state.seed_comments)
-@Radium
+@connect((state) => state.seed_comments)
 class ModerateCommentsSeed extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       showErrorDialogue: false,
-      showSuccessDialogue: false
+      showSuccessDialogue: false,
     };
   }
-  handleSubmitSeed () {
+  handleSubmitSeed() {
     const comment = {
       txt: this.refs.seed_form.value,
       pid: "mypid",
       conversation_id: this.props.params.conversation_id,
       // vote: 0,
-      is_seed: true
+      is_seed: true,
     };
-    this.props.dispatch(handleSeedCommentSubmit(comment))
+    this.props.dispatch(handleSeedCommentSubmit(comment));
+    this.refs.seed_form.value = "";
```

Comment:
Ditch this and bring back the logic that clears textarea only on success. Here's the diff:

```diff
diff --git a/client-admin/src/components/conversation-admin/seed-comment.js b/client-admin/src/components/conversation-admin/seed-comment.js
index de52d291..0779682f 100644
--- a/client-admin/src/components/conversation-admin/seed-comment.js
+++ b/client-admin/src/components/conversation-admin/seed-comment.js
@@ -27,7 +27,6 @@ class ModerateCommentsSeed extends React.Component {
       is_seed: true,
     };
     this.props.dispatch(handleSeedCommentSubmit(comment));
-    this.refs.seed_form.value = "";
   }
   handleTextareaChange(e) {
     this.props.dispatch(seedCommentChanged(e.target.value));
@@ -69,6 +68,7 @@ class ModerateCommentsSeed extends React.Component {
               borderColor: "mediumGray",
             }}
             onChange={this.handleTextareaChange.bind(this)}
+            value={this.props.seedText}
             maxLength="400"
             ref="seed_form"
           />
@@ -83,5 +83,3 @@ class ModerateCommentsSeed extends React.Component {
 }
 
 export default ModerateCommentsSeed;
-
-// value={this.props.seedText}
```

And a better demonstration of what i mean:

![screencast](http://g.recordit.co/BbhH8ynnLr.gif)